src/components/WhatsNew.vue

```diff
@@ -73,47 +114,75 @@
 </script>
 
 <style lang="scss">
-$darkGreen: #1F7564;
 $nearBlack: #181a1a;
-
+$coolBlue: rgb(66, 145, 235);
  // mobile layout
 #viz-latest {
     display:grid;
     width: 100%;
     height: auto;
```

Comment:
This grid seems a little taller than necessary right now -- making for a bit of a large gap before the Visualizations section:
![image](https://user-images.githubusercontent.com/54007288/140168438-9ea731ff-7db8-4632-b324-a20eed838269.png)